.github/workflows/rust.yml

```
@@ -52,8 +52,20 @@ jobs:
         override: true
     - name: build
       run: cargo build -v --no-default-features --features="avif"
-    # We don't test this yet, as it requires libdav1d which isn't in ubuntu 20.04
-    # run: cargo build -v --no-default-features --features="avif-decoding"
+  test_avif_decoding:
+    runs-on: ubuntu-20.04
+    steps:
+    - name: install-dependencies
+      run: sudo apt update && sudo apt install ninja meson
```

Ops, I had forgotten that the apt package is under a different name :sweat_smile: 

```suggestion
      run: sudo apt update && sudo apt install ninja-build meson
```